sticht/rollbacks/sources/splunk.py

```diff
@@ -18,7 +18,7 @@
 log = logging.getLogger(__name__)
 
 DEFAULT_SPLUNK_POLL_S = 1
```

Comment:
I suspect this isn't being used anywhere. That's probably on me forgetting to remove it after switching to oneshot searches